Remotion/ObjectBinding/Web/Res/HTML/BocAutoCompleteReferenceValue.jquery.js

```diff
@@ -1445,7 +1445,10 @@
         };
     };
 
-    $.Autocompleter.Selection = function(field, start, end) {
+    $.Autocompleter.Selection = function(field, start, end, focusInputAfterSelection) {

```

Comment:
It's funny, there is just one path that actually leads to the {{focusInputAfterSelection}} parameter being true ({{acceptInput}} when clicking the drop down button. And yet, due to the list of conditions until this point is reached, it doesn't seem like a good idea to just set the focus early on before doing all those nested calls.